src/components/Uploader/styles.js

```diff
@@ -79,12 +80,20 @@ export const PreviewActions = styled.div`
   bottom: -1px;
   border-radius: ${theme.borderRadius.s};
   background: ${theme.color.element.overlay};
-  padding: 0 ${theme.space.m};
-  text-align: right;
-  opacity: 0;
   pointer-events: none;
+  opacity: 0;
   z-index: 15;
-  transition: opacity 0.1s ease-out;
+`;
+
+export const TrashButton = styled(IconButton)`
+  z-index: 20;
+  margin: ${theme.space.m};
+  align-self: end;
+  padding: ${theme.space.s} !important;
+  background-color: #ffffff !important;
+  * {
+    color: ${({ iconColor }) => iconColor || theme.color_v3.feedback.error};
+  }
 `;
```

Comment:
🌷 Just so you know, as this is a trash button, you could include the icon in it's definition so that the caller doesn't have to set it. 
```suggestion
export const TrashButton = styled(IconButton).attrs({ icon: IconButton.ICONS.IconTrash })`
  z-index: 20;
  margin: ${theme.space.m};
  align-self: end;
  padding: ${theme.space.s} !important;
  background-color: #ffffff !important;
  * {
    color: ${({ iconColor }) => iconColor || theme.color_v3.feedback.error};
  }
`;
```

Reply:
Not sure I understand your point here 🙈  Is it better practice to set up the icon in style rather than in the component itself in index.js ?

Reply:
Not always but in this case, will you ever call `TrashButton` with another icon than the trashcan? Probably not, so it makes sense to make the icon baked in the component, as a default value, so that when you use `TrashButton` all you really need to provide is an `onClick` method.

Reply:
Ok thanks for the explanation, I change it with the icon directly setup in style :)

Reply:
Ok for u @ganlhi ? :)

Reply:
Yes, looks good 👍🏻